packages/gnome_sudoku.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gnome_sudoku < Package
+class Gnome_sudoku < Meson
   description 'Sudoku puzzle game for GNOME'
   homepage 'https://wiki.gnome.org/Apps/Sudoku'
   @_ver = '3.38.0'
```

Comment:
Update to [46.0](https://download.gnome.org/sources/gnome-sudoku/46/gnome-sudoku-46.0.tar.xz)?